additional information t of the Azure Compute Gallery (#1058)
MicrosoftDocs/cloud-adoption-framework: docs/scenarios/wvd/eslz-business-continuity-and-disaster-recovery.md

  - Make sure that the managed disk for the local VM is large enough to accommodate the local cache of all users' FSLogix profile and Office containers.

- Use Azure Compute Gallery to replicate golden images to different regions.

>   - Golden images do not participate in providing users the ability to connect to their session host VM; however, they play a critical role in how quickly you are able to run the provisioning process of new virtual machines on a host pool and therefore must be backed up and available.

Reviewer: ```suggestion
  - Golden images don't participate in providing users the ability to connect to their session host VM. However, they play a critical role in how quickly you are able to run the provisioning process of new virtual machines on a host pool and therefore must be backed up and available.
```
